test/src/nyc-test.js

```diff
@@ -405,7 +421,7 @@ describe('nyc', function () {
       })
       var report = reports[0]['./test/fixtures/not-loaded.js']
 
-      reports.length.should.equal(1)
+      reports.length.should.equal(enableCache ? 2 : 1)
```

Comment:
I had to add this because the tests don't clean up after themselves between runs.
They really should.
After we merge this, we should just add `rimraf .... all the output dirs...` to the top of the test file. That acts like a `beforeEach` will get us a clean state for every test.

If that breaks a test because it needed a specific output from a previous test, it should instead explicitly spawn the CLI. That makes our tests much more explicit.